test/ctr.bats

```diff
@@ -227,6 +228,7 @@ function teardown() {
 	run crictl start "$ctr_id"
 	echo "$output"
 	[ "$status" -eq 0 ]
+	wait_until_exit "$ctr_id"
 	run crictl stop "$ctr_id"
 	echo "$output"
 	# Ignore errors on stop.
```

Comment:
Now that we're waiting for a clean exit, we can drop the `stop` and `inspect` calls and go straight to `rm` (like we do in master [here][1]).  We can also make the same change to a number of other places in `ctr.bats` (and likely other files, although I haven't checked).  I'm fine with some/all of that getting punted into follow-up PRs if you don't want to handle it here.

[1]: https://github.com/kubernetes-incubator/cri-o/blob/f9f8a53f1a0816c98e65606e674232d11319cc7f/test/ctr.bats#L80-L83

Reply:
fixed although not sure if I made the changes in all `ctr.bats`